apiary.apib

```diff
@@ -2048,6 +2048,16 @@ Draft a new invoice.
                         + percentage
                 + description: `winter promotion` (string, required)
         + note: `Invoice comments` (string, optional)
+        + expected_payment_method (object, optional)
+            + method: `sepa_direct_debit` (enum[string], required)
+                + Members
+                    + `sepa_direct_debit`
+                    + `credit_card`
+            + initiator: `supplier` (enum[string], required)
```

Comment:
the required initiator feels a bit confusing to me. Can you do `sepa_direct_debit` with the initiator set to `customer`?

Reply:
The initiator is the person who initiates the payment itself. For sepa direct debit you can have a supplier initiated payment, which means you already have a mandate from that customer to let him pay automatically. But you can also expect him to pay with sepa direct debit when you don't have the mandate yet. In that case,  the customer still needs to authorise the payment, by giving a one-off mandate for instance. Both use cases could be supported by a payment integrator, so we need to be able to provide this information to them. 

The same usecases are applicable for creditcard. If you expect someone to pay with creditcard, it does not mean that the payment will happen automatically. In Italy, you need to state which payment method you expect in order to generate an electronic invoice. Not all of these payment methods are supplier initiated (ie. automatic).

Reply:
Ok, that makes it more clear. Might be confusing for users reading our documentation too though. Would it be possible to not make `initiator` required? So that you don't have to figure out what it means if you don't want to use it?